internal/services/postgres/postgresql_flexible_server_configuration_resource.go

```diff
@@ -81,9 +79,6 @@ func resourceFlexibleServerConfigurationUpdate(d *pluginsdk.ResourceData, meta i
 	locks.ByName(id.FlexibleServerName, postgresqlFlexibleServerResourceName)
 	defer locks.UnlockByName(id.FlexibleServerName, postgresqlFlexibleServerResourceName)
 
-	locks.ByName(id.ConfigurationName, postgresqlFlexibleServerConfigurationResourceName)
-	defer locks.UnlockByName(id.ConfigurationName, postgresqlFlexibleServerConfigurationResourceName)
-
```

Comment:
These were added recently in #27355 and seem to have fixed an issue so I don't think we should be removing them

Reply:
Could you resolve the merge conflicts so this can be merged?